projects/plugins/boost/app/assets/src/js/pages/connection/Connection.svelte

```diff
@@ -14,10 +13,6 @@
 		__( 'Improve your SEO ranking', 'jetpack-boost' ),
 		__( 'Sell more stuff', 'jetpack-boost' ),
 	];
-
-	$: if ( $connection.connected ) {
-		onConnectionComplete();
-	}
```

Comment:
I assume this is removed because it's handled in `initialize` in `connection.ts`? Nice. 🧹